testnet/stacks-node/src/config.rs

```diff
@@ -105,13 +136,20 @@ mod tests {
 }
 
 impl ConfigFile {
-    pub fn from_path(path: &str) -> ConfigFile {
-        let content_str = fs::read_to_string(path).unwrap();
-        Self::from_str(&content_str)
+    pub fn from_path(path: &str) -> Result<ConfigFile, String> {
+        let content_str = fs::read_to_string(path);
+        if content_str.is_err() {
+            return Err(format!("Invalid path"));
```

Comment:
Do you want to capture the error returned by `fs::read_to_string()` in the error message?  Maybe something like this?

```rust
let content_str = match fs::read_to_string(path) {
   Ok(s) => s,
   Err(e) => {
      return Err(format!("Invalid path: {:?}", &e))
    }
};
```

(I haven't compiled the above; you might be able to get away with the less-verbose `"{}"` formatter instead of `"{:?}"`).

Reply:
could also do something like `let content_str = fs::read_to_string(path).map_err(|e| Err(format!("Invalid path: {:?}", &e)))?;`